doc/manual/installing.rst

```diff
@@ -3,21 +3,25 @@ Installing ARTIQ
 
 ARTIQ can be installed using the Nix (on Linux) or MSYS2 (on Windows) package managers. Using Conda is also possible on both platforms but not recommended.
 
-.. _installing-nix-users:
-
 Installing via Nix (Linux)
 --------------------------
 
-First, install the Nix package manager. Some distributions provide a package for the Nix package manager, otherwise, it can be installed via the script on the `Nix website <http://nixos.org/nix/>`_. Make sure you get Nix version 2.4 or higher.
+First install the Nix package manager. Some distributions provide a package for it; otherwise, it can be installed via the script on the `Nix website <http://nixos.org/nix/>`_. Make sure you get Nix version 2.4 or higher. Prefer a single-user installation for simplicity. 
 
-Once Nix is installed, enable Flakes: ::
+Once Nix is installed, enable flakes, for example by running: ::
 
   $ mkdir -p ~/.config/nix
   $ echo "experimental-features = nix-command flakes" > ~/.config/nix/nix.conf
```

Comment:
I know it is not in the changes, but I think `>>`  instead of  `>` would be more suitable here, so that it appends, and not overwrites.